### PR TITLE
C file for Fishing, some changes to collider scripts, a few minor cleanups

### PR DESCRIPTION
**Pull request by EllipticEllipsis**
Before opening this PR, ensure the following:
- `./format.sh` was run to apply standard formatting.
- `make` successfully builds a matching ROM.
- No new compiler warnings were introduced during the build process.
    - Can be verified locally by running `tools/warnings_count/check_new_warnings.sh`
- New variables & functions should follow standard naming conventions.
- Comments and variables have correct spelling.
---

- Added Fishing to the actor list, split it properly.
- Made some minor changes to `colliderinit.py` (it now outputs `D_address`s instead of `sCylinderInit` etc., and puts the likely eventual name on the line above: this should make it simpler to compile the C initially and then swap over later. (I wouldn't be too concerned about this change: it's intended to render the script obsolete shortly by importing the colliders, and most of the rest of the Init data, shortly anyway.)
- Improved `actor_symbols.py` to throw a clear exception rather than cryptic messages, especially when called in other scripts.
- Added a script to disassemble `CollisionCheckInfoInit`s
- Various minor cleanups in actors I saw while doing all of the above.

### CODE REVIEW

File: src/overlays/actors/ovl_En_Fishing/overlay.cfg [hunk 1]
```diff
@@ -0,0 +1,2 @@
+ovl_En_Fishing
+z_en_fishing.c
```
Comment (Kenix3) [inline src/overlays/actors/ovl_En_Fishing/overlay.cfg:2]: missing trailing newline

**Review by Kenix3 — CHANGES_REQUESTED** [review]

File: src/overlays/actors/ovl_En_Fishing/z_en_fishing.c [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+#define THIS ((EnFishing*)thisx)
+
+void EnFishing_Init(Actor* thisx, GlobalContext* globalCtx);
+void EnFishing_Destroy(Actor* thisx, GlobalContext* globalCtx);
+void EnFishing_Update(Actor* thisx, GlobalContext* globalCtx);
+void EnFishing_Draw(Actor* thisx, GlobalContext* globalCtx);
+
+/*
+const ActorInit En_Fishing_InitVars = {
+    0x0079,
```
Comment (AngheloAlf) [inline src/overlays/actors/ovl_En_Fishing/z_en_fishing.c:20]:
```suggestion
    ACTOR_EN_FISHING,
```

File: tools/overlayhelpers/colliderinit.py [hunk 3]
```diff
@@ -397,7 +404,7 @@ def GetColliderInit(address, type, num):
     if fileResult[0] is None:
         return("Invalid address")
 
-    print(hex(fileResult[1]), "in", fileResult[0].split(os.sep)[-1])
+    # print(hex(fileResult[1]), "in", fileResult[0].split(os.sep)[-1])
```
Comment (AngheloAlf) [inline tools/overlayhelpers/colliderinit.py:407]: Is this line supposed to be commented?

Comment (EllipticEllipsis) [reply tools/overlayhelpers/colliderinit.py:407]: Hm. I think I'll delete the previous comment (which is superseded by the error message in `actor_symbol`), but keep this line: it is a useful check that you're in the right file, if nothing else.

**Review by AngheloAlf — APPROVED** [review]

**Review by Kenix3 — APPROVED** [review]

**Pull request merged**